spellbook/models/nft/nft_trades.sql

```diff
@@ -1,16 +1,17 @@
 {{ config(
         alias ='trades',
-        materialized ='incremental',
-        file_format ='delta',
-        incremental_strategy='merge',
-        unique_key='unique_trade_id'
+        materialized ='incremental'
         )
 }}
 
-SELECT blockchain, project, version, tx_hash, block_time, amount_usd, amount, token_symbol, token_address, unique_trade_id FROM 
-(SELECT blockchain, project, version, tx_hash, block_time, amount_usd, amount, token_symbol, token_address, unique_trade_id FROM {{ ref('opensea_trades') }} 
-UNION ALL
-SELECT blockchain, project, version, tx_hash, block_time, amount_usd, amount, token_symbol, token_address, unique_trade_id FROM {{ ref('magiceden_trades') }}) 
+SELECT *
+FROM 
+(
+SELECT * FROM {{ ref('opensea_trades') }} 
+        UNION ALL
```

Comment:
reminder. I think this should be union not union all. just calling out so it doesn't slip through.

Reply:
Done :) Thanks for your review !